app/_src/policies/meshhttproute.md

```diff
@@ -462,15 +478,17 @@ spec:
                 requestMirror:
                   percentage: 30
                   backendRef:
-                    kind: MeshServiceSubset
-                    name: backend_kuma-demo_svc_3001
-                    tags:
-                      version: v1_experimental
+                    kind: MeshService
```

Comment:
I am not sure what to do with this, the example generated from it when not using MeshService in incorrect. There is the same example in MeshTCPRoute. For old scenario with `kuma.io/service` we should use MeshServiceSubset and with tags

Reply:
We should fix the generator code right? Can we if_version it for now, create an issue to fix it and merge this?

Reply:
oh, it's in `backendRef` - maybe @lobkovilya has an idea how to fix it

Reply:
fixed this on https://github.com/kumahq/kuma-website/pull/2056

Reply:
it was fixed with https://github.com/kumahq/kuma-website/pull/2056 I will update examples